root/user/UserProfile.js

```diff
@@ -341,6 +341,8 @@ const UserProfileStatistics = withCatalystContext(({
 }: UserProfileStatisticsProps) => {
   const voteTotals = votes.pop();
   const encodedName = encodeURIComponent(user.name);
+  const allAppliedCount = Number(editStats.accepted_count) +
+                           Number(editStats.accepted_auto_count);
 
```

Comment:
These are defined as `number` in `EditStatsT`, so the type must be wrong (which isn't ideal).

Does it help if we change https://github.com/metabrainz/musicbrainz-server/blob/e1886bf/lib/MusicBrainz/Server/Data/Editor.pm#L582-L582 to do `$count + 0` instead? That should cause the JSON serializer to produce actual numbers, hopefully.

Reply:
I'm not sure why I was doing this in the first place. I checked, and even looking at a just-created user with 0 edits, everything seems to just work.

Reply:
I'm not sure why I was doing this in the first place. I checked, and even looking at a just-created user with 0 edits, everything seems to just work.